pkg/util/exec/exec.go

```diff
@@ -177,6 +177,7 @@ func (*CommandExecutor) ExecuteCommandWithOutputFileTimeout(timeout time.Duratio
 }
 
 // ExecuteCommandWithOutputFile executes a command with output on a file
+// #nosec G307 Calling defer to close the file without checking the error return is not a risk for a simple file open and close
```

Comment:
and to be clear for others, there is a separate work item to follow up and call `Close()` as described [here](https://github.com/rook/rook/pull/5892#discussion_r461080627).